src/Microsoft.AspNetCore.Diagnostics.HealthChecks/HealthCheckResponseWriters.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.Diagnostics.HealthChecks
+{
+    public static class HealthCheckResponseWriters
+    {
+        public static Task WriteMinimalPlaintext(HttpContext httpContext, CompositeHealthCheckResult result)
+        {
+            if (httpContext == null)
+            {
+                throw new ArgumentNullException(nameof(httpContext));
+            }
+
+            if (result == null)
+            {
+                throw new ArgumentNullException(nameof(result));
+            }
+
+            httpContext.Response.ContentType = "text/plain";
+            return httpContext.Response.WriteAsync(result.Status.ToString());
```

Comment:
`InvariantCulture`

Reply:
https://msdn.microsoft.com/en-us/library/3bkcbx85(v=vs.110).aspx - they actually want you to use the no-arg version.